dashboard/test/ui/features/learning_platform/level_types/level_group_multi_page_dots.feature

```diff
@@ -87,7 +84,6 @@ Scenario: Submit three pages as... 1. all, 2. none, 3. some questions answered.
 
   # Go to the course page and verify the same three dots.
   Then I navigate to the course page for "allthethings"
-  And I wait until jQuery Ajax requests are finished
   And I verify progress for lesson 23 level 2 is "perfect_assessment"
   And I verify progress for lesson 23 level 3 is "not_tried"
```

Comment:
If we have multiple `verify progress` statements in a row, should subsequent statements be run with `without waiting`?

Reply:
Good question.  This implementation of `verify progress` is only inefficient if we have several `verify progress` statements in a row that check for "not_tried".  Checking for all other statuses will complete immediately if progress data is already loaded.  So my recommendation is to treat `without waiting` as an advanced optimization -- we shouldn't need to use it except in unusual cases where we check many bubbles for "not_tried" in a row (such as in lesson_lock.feature).